.github/workflows/tenzir.yaml

```diff
@@ -718,6 +726,11 @@ jobs:
           ccache --show-stats
           # Print statistics about cache compression.
           ccache --show-compression
+      - name: Save ccache Cache
+        uses: actions/cache/save@v3
+        with:
+          path: ${{ env.CCACHE_DIR }}
+          key: ${{ github.workflow }}-${{ matrix.tenzir.name }}-${{ matrix.tenzir.compiler }}-${{ needs.configure.outputs.head-ref-slug }}-${{ github.sha }}
```

Comment:
Would it be possible to de-dup the value for `key`, to prevent it going out of sync?

Reply:
It's actually a bit more complicated, because key matched here will almost never be the primary key but one of the prefix-matched restore keys. Additionally the path has to match as well as one of the keys. I'll just keep it for now and hope that no one will dare to touch it again :D